plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -146,7 +146,6 @@ protected QueryRunner createQueryRunner()
     protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         switch (connectorBehavior) {
-            case SUPPORTS_COMMENT_ON_COLUMN:
```

Comment:
As a follow-up can you please add tests for CREATE TABLE with column comment?

see https://github.com/trinodb/trino/blob/dbc25acdb019f12108860a70c5dd699e006c16a8/testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java#L410-L413